### PR TITLE
Add generators tabular creations

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

**Review by EstherDarkish — COMMENTED** [review]
I have a NullPointerException in the logs when uploading a CSV. Ask me and I'll provide it for testing.

**Comment by FranckLecuyer** [comment]
> I have a NullPointerException in the logs when uploading a CSV. Ask me and I'll provide it for testing.

Fixed in front PR (connection direction field pb)

**Review by EstherDarkish — APPROVED** [review]
Test OK Code OK

File: src/main/java/org/gridsuite/modification/server/dto/TabularCreationInfos.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@Schema(description = "Tabular creation")
+@JsonTypeName("TABULAR_CREATION")
+@ModificationErrorTypeName("TABULAR_CREATION_ERROR")
+public class TabularCreationInfos extends ModificationInfos {
+
+    @Schema(description = "Creation type")
+    @NonNull
+    private ModificationType creationType;
+
+    @Schema(description = "creations")
```
Comment (Meklo) [inline src/main/java/org/gridsuite/modification/server/dto/TabularCreationInfos.java:46]: Capitalize ?

Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/dto/TabularCreationInfos.java]: Done

**Review by Meklo — APPROVED** [review]

**Pull request merged**